src/meta/types/build.rs

```diff
@@ -47,91 +47,91 @@ fn build_proto() {
         .file_descriptor_set_path(out_dir.join("meta_descriptor.bin"))
         .type_attribute(
             "SeqV",
-            "#[derive(Eq, serde::Serialize, serde::Deserialize)]",
+            "#[derive(Eq, serde::Serialize, serde::Deserialize, deepsize::DeepSizeOf)]",
```

Comment:
Is there a performance issue with `deepsize::DeepSizeOf`? I'm not certain whether it's being called each time or not.

We encountered a "get-size" issue years ago:
https://github.com/datafuselabs/databend/pull/8461
I'm not certain if they're the same; this is just a friendly suggestion.

Reply:
To calculate the size of a 98KB raft log, it takes 1~2 us. It's only used when replication raft log. the performance is quite enough.